ThreeSum.java

```diff
@@ -0,0 +1,103 @@
+
+// Time Complexity : O(n^2logn)
+// Space Complexity : O(1)
+// Did this code successfully run on Leetcode :
+
+
+//******
+// Any problem you faced while coding this : Can you  please explain time complex and space Complex for the below solutions
```

Comment:
For sorting it will take n*logn + nested loop(for and while) which is doing in N^2 ---> Total TC : N^2 + N*LogN

Reply:
If we ignore TC for sorting, then it will be only N^2